HPXMLtoOpenStudio/resources/waterheater.rb

```diff
@@ -1085,7 +1085,8 @@ def self.add_desuperheater(model, runner, water_heating_system, tank, loc_space,
                                      model: model,
                                      runner: runner,
                                      ua: assumed_ua,
-                                     is_dsh_storage: true)
+                                     is_dsh_storage: true,
+                                     power_outage_periods: [])
```

Comment:
This doesn't seem like the best fix? If the `create_new_heater` can't accept a nil for power_outage_periods, then the method arguments should be changed from `power_outage_periods: nil` to `power_outage_periods: []` so that it has a default value that works. And then you don't need this change.